README.md

```diff
@@ -1080,3 +1081,74 @@ describe('yourModule', () => {
 **[⬆ back to summary](#-summary)**
 
 ---
+
+## 11. Typescript
+
+- 11.1 [Do not use any type](#111-do-not-use-any-type)
+- 11.2 [Naming convention](#112-naming-convention)
+- 11.3 [Exporting types](#113-exporting-types)
+- 11.4 [Types within a file](#114-types-within-a-file)
+
+### 11.1 Do not use any type
+
+Avoid using `any` type. It is better to use the type that is more specific.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Do you think it's worth mentioning instead  ``avoid any`` prefer use unknown when needs? At least it guarantees that will have to do the typing checks

Reply:
Very nice suggestion! Added in https://github.com/juntossomosmais/frontend-guideline/pull/28/commits/6753a9d429c1513af1e8ceeefd12ca9f0b6722ea